api_volontaria/apps/volunteer/templates/participation_confirmation.txt

```diff
@@ -0,0 +1,24 @@
+Bonjour {{FIRST_NAME}},
+
+Nous vous confirmons votre participation à l'événement "{{EVENT_NAME}}" comme {{TYPE_PARTICIPATION}}.
+
+* Activité: {{TASK_TYPE}}
+
+* Date: {{START_DATE}}
+* Heure de début: {{START_TIME}}
+* Heure de fin: {{END_TIME}}
+
+* Lieu:
+{{LOCATION_NAME}}
+{{ADDRESS_LINE_1}}
+{{ADDRESS_LINE_2}}
+{{POSTAL_CODE}}
+{{CITY}}
+{{STATE_PROVINCE}}
+
+Au plaisir de vous y voir.
+
+# Comment insérer le nom de l'OSBL?
+** L'ÉQUIPE XXXXXXXX
```

Comment:
In **settings.py** you can find a variable of configuration for that:
https://github.com/Volontaria/API-Volontaria/blob/f5ccbe50bd1401b3ad7cff2447e69d8702c24d2e/api_volontaria/settings.py#L252-L264

You can use the settings in the models by doing something like:

```python
from django.conf import settings

settings.LOCAL_SETTINGS['ORGANIZATION']
```